go/src/koding/workers/removenonexistents/main.go

```diff
@@ -0,0 +1,371 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"koding/db/models"
+	helper "koding/db/mongodb/modelhelper"
+	"koding/helpers"
+	"koding/tools/config"
+	"os"
+	"strings"
+
+	mgo "gopkg.in/mgo.v2"
+
+	"github.com/koding/cache"
+)
+
+var (
+	conf        *config.Config
+	flagProfile = flag.String("c", "prod", "Configuration profile from file")
+	flagSkip    = flag.Int("s", 0, "Configuration profile from file")
+	flagLimit   = flag.Int("l", 1000, "Configuration profile from file")
+	flagDry     = flag.Bool("dry", false, "dry run")
+	flagColls   = flag.String("colls", "jUsers,jAccounts,jWorkspaces,jNames,jComputeStacks,jCombinedAppStorages,relationships", "collections to clean up")
```

Comment:
IDE part can be removed but we need to have a similar collection to keep collaboration data, currently it's the only requirement for this collection from IDE perspective.